### DISCUSSION TITLE
bind_content_from in ui.code

### DISCUSSION THREAD

**Discussion by JeanBaptiste-dlb** [zauberzeug/nicegui#3077] [Ideas / Feature Requests]
Hey there, 

Thanks for the great work.

I believe it would be nice to be able to modify the content of ui.code based on user interaction. With something similar to bind_content_from

**Comment by rodja** [comment]
We recently introduced `ui.codemirror` for this: https://nicegui.io/documentation/codemirror
Does that work for you?

**Comment by falkoschindler** [comment]
Apart from the possibility of using `ui.codemirror`, a bindable content property for `ui.code` sounds like a very reasonable feature.
So I checked how hard it would be to implement, which resulted in PR #3089. 😬

**Reply by JeanBaptiste-dlb** [reply]
Thank you for the feature implementation
